hack/scripts/minikube-install.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+# Copyright 2024 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
Need to add some documentation about how to get up and running with this.

Reply:
I think the only pre-req should be having a working virutalization setup on the device you're running this on. Minikube prints very useful errors for this.

Reply:
Also having the docker CLI and the kubectl CLI.